src/ReleaseHistory.md

```diff
@@ -1,6 +1,6 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
-## **v3.2.0** (UNRELEASED)
+## **v4.0.0-rc1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/4.0.0-rc1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/4.0.0-rc1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/4.0.0-rc1) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/4.0.0-rc1) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/4.0.0-rc1)
```

Comment:
>** [](http://example.com/codeflow?start=3&length=2)

What's going on here exactly? Why do we have an RC?

We need to pick a version (not pre-release) and ship it.

It can be 3.2 or it can be 4.0 (due to the large number of breaking changes).

In general, let's not create multiple PRs for package release. Create one and then let the review process tell you what's right.

@EasyRhinoMSFT, what's the call? 3.2 or 4.0?

Reply:
Got it. I see his reply with 4.0, I will update it now.

Reply:
I don't think there is anything with this that justifies a release candidate version. The SDK version is unrelated to BinSkim. As Michael noted this morning, what's in main is always release quality.

Reply:
Sounds good, I'm happy with 4.0, there's a lot of update to this one. :)